rspec-expectations/lib/rspec/matchers/built_in/change.rb

```diff
@@ -286,8 +286,8 @@ def to(value)
         # @private
         def does_not_match?(event_proc)
           if @description_suffix
-            raise NotImplementedError, "`expect { }.not_to change { }.to()` " \
-                                       "is not supported"
+            raise NotImplementedError, "Using `expect { }.not_to change { }` " \
+                                       "with the `to()` matcher is not supported."
```

Comment:
This should actually say `from()`, shouldn't it? Apparently this was a copy-pasta.

Reply:
ah, nice catch, I dug a little into this and found out that it should indeed be `to()`, but the `to()` has to be after a `from()`

e.g.

Supported:
`expect {}.not_to change {}.from(5)`

Not supported:
`expect {}.not_to change {}.from(5).to(6)`

>  Using `expect { }.not_to change { }.from()` with the `to()` matcher is not supported.

Reply:
I'm open to suggestions for the error message as well - I came up with it on the fly

Reply:
What you say makes sense. There's nothing wrong with `expect { }.not_to change { }.from()`, we're just explicitly setting an expectation for the initial value, and expect it not to change.

It turns out that the code responsible for `change { }.from().to()` is different from the one responsible for `change { }.to().from()`, and I can't quickly grasp why this is done like this.

But this whole thing is orthogonal to your change, so please never mind the above.

Reply:
See #178